packages/waku/src/lib/vite-rsc/deploy/aws-lambda/entry.ts

```diff
@@ -29,4 +29,4 @@ export const handler: any = import.meta.env.WAKU_AWS_LAMBDA_STREAMING
   ? honoAwsLambda.streamHandle(app)
   : honoAwsLambda.handle(app);
 
-export { handleBuild } from '../../build.js';
+export { processBuild } from '../../build.js';
```

Comment:
@hi-ogawa I'm not sure how this export is used. Could you explain?

Reply:
The code is here. It imports `dist/server/index.js` after the build and runs `handleBuild` export.

https://github.com/wakujs/waku/blob/d3d84cf94cc360c103795172f092b3e64f4f150a/packages/waku/src/lib/vite-rsc/plugin.ts#L394-L409

Reply:
Thanks! And, I missed `pathToFileURL(entryPath).href`... Yes, it makes totally sense.